tests/test_optics.py

```diff
@@ -386,29 +374,31 @@ def test_OpticalPSF_aberrations_struts():
     # check anyway...)
     optics_2 = galsim.OpticalPSF(
         lod, obscuration=obscuration, nstruts=5, strut_thick=0.04, strut_angle=0.*galsim.degrees,
-        astig2=0.04, coma1=-0.07, defocus=0.09, oversampling=1)
+        astig2=0.04, coma1=-0.07, defocus=0.09)
     myImg = optics.drawImage(myImg, scale=0.2*lod, use_true_center=True, method='no_pixel')
+
+    # This test failed at precision=6 when switching to the new optics framework (for 0.04% of
+    # pixels).  However, upon investigation, it appears that the differences are due the
+    # illumination of 3 extra aperture pixels under the old framework compared to the new framework.
+    # When manually zeroing these pixels in the old framework and reproducing a PSF, the test
+    # passes.  So here, we simply degrade the precision to 5.
```

Comment:
So I'm not sure what happened, but apparently the old versions of the optics comparison files *do* work with the reimplemented OpticalPSF.  So I restored them.